src/QMCWaveFunctions/LCAO/SoaLocalizedBasisSet.h

```diff
@@ -172,6 +172,24 @@ class SoaLocalizedBasisSet : public SoaBasisSetBase<ORBT>
    * @param aos a set of Centered Atomic Orbitals
    */
   void add(int icenter, std::unique_ptr<COT> aos);
+
+
+  /** initialize a shared resource and hand it to collection
+   */
+  void createResource(ResourceCollection& collection) const override;
+
+  /** acquire a shared resource from collection
+   */
+  void acquireResource(ResourceCollection& collection,
+                       const RefVectorWithLeader<SoaBasisSetBase<ORBT>>& spo_list) const override;
+
+  /** return a shared resource to collection
+   */
+  void releaseResource(ResourceCollection& collection,
+                       const RefVectorWithLeader<SoaBasisSetBase<ORBT>>& spo_list) const override;
```

Comment:
Should it be basisset_list instead of spo_list?

Reply:
fixed